### PR TITLE
Fixes uplink.dm not accepting null values for restricted species

### PR DESCRIPTION
**Pull request by CRTXBacon**
Pretty important PR.

:cl: CRTXBacon
fix: Fixes an error caused by the species restricted category in uplinks
/:cl:

### CODE REVIEW

**Comment by k-k-karma** [comment]
HOLY FUCKING WEBEDIT BATMAN

**Comment by CRTXBacon** [comment]
Wasn't even a webedit (pasting one line was quicker than uploading though), was just in a rush to get this out.

**Comment by k-k-karma** [comment]
WHY ARE YOU UPLOADING FILES WITH GITHUB

File: code/datums/components/uplink.dm [hunk 1]
```diff
@@ -147,7 +147,7 @@ GLOBAL_LIST_EMPTY(uplinks)
 							is_inaccessible = 0
 					if(is_inaccessible)
 						continue
-				if(I.restricted_species.len)
+				if(I.restricted_species != null)
```
Comment (k-k-karma) [inline code/datums/components/uplink.dm]:
```suggestion
				if(length(I.restricted_species))
```

Comment (CRTXBacon) [reply code/datums/components/uplink.dm]: Has the same end result in just as many characters.

Comment (Barhandar) [reply code/datums/components/uplink.dm]: Also it should be LAZYLEN if you expect the list to ever be null.

Comment (ExcessiveUseOfCobblestone) [reply code/datums/components/uplink.dm]:
```suggestion
				if(I.restricted_species)
```

lol

Comment (CRTXBacon) [reply code/datums/components/uplink.dm]: oh yeah that's a thing that works, thanks cobby

**Pull request merged**